### DISCUSSION TITLE
Berry startup question...

### DISCUSSION THREAD

**Discussion by trlafleur** [arendst/Tasmota#14561] [Q&A] [answered]
I'm new to Berry, I found discussions --> 13975 to be very helpful in getting started...
but...

1) if I have a sensor defined in Tasmota, is there a way from Berry to stop it from sending MQTT during the telaperiod?
2) Then I want to set up a timer or ? to query the sensor and send the MQTT in my time frame.

**Comment by barbudor** [comment, marked as answer]
You can set Teleperiod to 0 to totally disable Teleperiod but that will also disable the STATE message (not just sensor)

Wiht `tasmota.read_sensors( [show_sensor:bool] ) -> string` you can query the json of the sensors and convert it to map.
Then republish what you want with `tasmota.publish( topic:string, payload:string[, retain:bool, start:int, len:int]) -> nil`
